src/components/forms/AddEmployeeForm.js

```diff
@@ -69,23 +118,19 @@ function AddEmployeeForm({ visible, onCreate, onCancel }) {
         >
           <Select size="large" placeholder="Select Employee Role">
             {/* Could be dynamic by mapping through list of roles */}
-            <Select.Option value="Administrator">Administrator</Select.Option>
-            <Select.Option value="Program Manager">
-              Program Manager
-            </Select.Option>
-            <Select.Option value="Service Provider">
-              Service Provider
-            </Select.Option>
+            <Select.Option value={1}>Administrator</Select.Option>
+            <Select.Option value={2}>Program Manager</Select.Option>
+            <Select.Option value={3}>Service Provider</Select.Option>
             ))
           </Select>
         </Form.Item>
-        <Form.Item label="Programs" name="programs">
+        {/* <Form.Item label="Programs" name="programs">
```

Comment:
Seems like connecting the Providers with the Programs could be useful for Admin. Keeping this code could save some time later on if that does get implemented on the BE. 
Perhaps a comment explaining this could keep the next group from deleting on sight?